README.md

```diff
@@ -1,241 +1,331 @@
 
-  If not, add it with `sudo adduser $USER render`, logout and log back in, and check
-  again.
+  ```bash
+  mkdir neo
+  cd neo
+  wget https://github.com/intel/compute-runtime/releases/download/22.14.22890/intel-gmmlib_22.0.2_amd64.deb
+  wget https://github.com/intel/intel-graphics-compiler/releases/download/igc-1.0.10840/intel-igc-core_1.0.10840_amd64.deb
+  wget https://github.com/intel/intel-graphics-compiler/releases/download/igc-1.0.10840/intel-igc-opencl_1.0.10840_amd64.deb
+  wget https://github.com/intel/compute-runtime/releases/download/22.14.22890/intel-opencl-icd_22.14.22890_amd64.deb
+  dpkg -i *.deb  # requires root permissions
+  apt-get install -y ocl-icd-libopencl1  # requires root permissions
+  cd ../ && rm -Rf neo
+  ```
 
-- Install recent versions of the [runtime libraries](https://github.com/intel/compute-runtime/releases) (not yet available
-  as conda packages)
+</details>
 
-For more in-depth information, you can refer to the [guides for system configuration for Intel hardware and software](https://www.intel.com/content/www/us/en/developer/articles/system-requirements/intel-oneapi-dpcpp-system-requirements.html) and to the discussions in this github issue https://github.com/IntelPython/dpnp/issues/1149
+- **oneAPI level zero loader**: alternatively, or in addition, the oneAPI level zero
+  backend can be used. This backend is more experimental, and is sometimes preferred
+  over opencl. Source and `deb` archives are available
+  [here](https://github.com/oneapi-src/level-zero/releases).
 
+###### Give permissions to submit GPU workloads
 
-The `dpctl.lsplatform()` can also list version informations on your SYCL
-runtime environment:
+Non-root users might lack permission to access the GPU device to submit workloads. Add
+the current user to the `video` group and/or `render` group:
 
+```bash
+sudo usermod -a -G video $USER
+sudo usermod -a -G render $USER
 ```
-python -c "import dpctl; dpctl.lsplatform()"
-```
-
-##### Install numba-dpex from source (2/2)
 
-Install and activate the Intel oneAPI DPC++ compiler shipped with the [Intel oneAPI base toolkit](https://www.intel.com/content/www/us/en/developer/tools/oneapi/base-toolkit-download.html).
+##### Setup a conda environment for numba-dpex (2/2)
 
-For instance, in Ubuntu, once the apt repo has been configured:
+You can setup a [`conda`](https://docs.conda.io/en/latest/) environment, and install
+dependencies (`numba-dpex` and `intel::dpcpp_linux-64`) distributed on the
+`conda-forge`, `intel` and experimental `dppy/label/dev` channels with:
 
-```
-sudo apt update
-sudo apt install intel-basekit
-source /opt/intel/oneapi/compiler/latest/env/vars.sh
+```bash
+export CONDA_DPEX_ENV_NAME=my-dpex-env
+conda create --yes --name $CONDA_DPEX_ENV_NAME \
+             --channel dppy/label/dev \
+             --channel conda-forge \
+             --channel intel \
+             numba-dpex intel::dpcpp_linux-64
```

Comment:
When running it, I get:
```
Looking for: ['numba-dpex', 'intel::dpcpp_linux-64', 'pip']

dppy/label/dev/noarch                                         No change
dppy/label/dev/linux-64                                       No change
intel/linux-64                                                No change
intel/noarch                                                  No change
intel/noarch                                                  No change
intel/linux-64                                                No change
conda-forge/noarch                                  10.6MB @   4.0MB/s  2.7s
conda-forge/linux-64                                28.3MB @   4.0MB/s  7.3s
Encountered problems while solving:
  - package dpcpp_linux-64-2022.2.0-intel_8734 is excluded by strict repo priority
  - package numba-dpex-0.18.1-py39hfc4b9b4_33 requires dpnp >=0.10.1, but none of the providers can be installed
```

Do you think this is addressable?

Reply:
I use a virgin Ubuntu container to test the guide. I tested both with mamba and conda, using the latest `Mambaforge` `sh` installer. I can't reproduce your error.  Most likely something is different on your side (conda/mamba version, some specific resolution config,...)

Here's how it resolves for me:

<details>

```
  + _libgcc_mutex               0.1  conda_forge          conda-forge/linux-64        Cached
  + _openmp_mutex               4.5  2_gnu                conda-forge/linux-64        Cached
  + bzip2                     1.0.8  h7f98852_4           conda-forge/linux-64        Cached
  + ca-certificates       2022.12.7  ha878542_0           conda-forge/linux-64        Cached
  + dpcpp-cpp-rt           2022.2.1  intel_16953          intel/linux-64              Cached
  + dpcpp_impl_linux-64    2022.2.1  intel_16953          intel/linux-64              Cached
  + dpcpp_linux-64         2022.2.1  intel_16953          intel/linux-64              Cached
  + dpctl                0.14.1dev0  py39h8c27c75_10      dppy/label/dev/linux-64     Cached
  + dpnp                     0.11.0  py39h2bc3f7f_20      dppy/label/dev/linux-64     Cached
  + icc_rt                 2022.2.1  intel_16953          intel/linux-64              Cached
  + icu                        70.1  h27087fc_0           conda-forge/linux-64        Cached
  + intel-cmplr-lib-rt     2022.2.1  intel_16953          intel/linux-64              Cached
  + intel-cmplr-lic-rt     2022.2.1  intel_16953          intel/linux-64              Cached
  + intel-opencl-rt        2022.2.1  intel_16953          intel/linux-64              Cached
  + intel-openmp           2022.2.1  intel_16953          intel/linux-64              Cached
  + ld_impl_linux-64           2.39  hcc3a1bd_1           conda-forge/linux-64        Cached
  + libblas                   3.9.0  16_linux64_openblas  conda-forge/linux-64        Cached
  + libcblas                  3.9.0  16_linux64_openblas  conda-forge/linux-64        Cached
  + libffi                    3.4.2  h7f98852_5           conda-forge/linux-64        Cached
  + libgcc-ng                12.2.0  h65d4601_19          conda-forge/linux-64        Cached
  + libgfortran-ng           12.2.0  h69a702a_19          conda-forge/linux-64        Cached
  + libgfortran5             12.2.0  h337968e_19          conda-forge/linux-64        Cached
  + libgomp                  12.2.0  h65d4601_19          conda-forge/linux-64        Cached
  + libhwloc                  2.8.0  h32351e8_1           conda-forge/linux-64        Cached
  + libiconv                   1.17  h166bdaf_0           conda-forge/linux-64        Cached
  + liblapack                 3.9.0  16_linux64_openblas  conda-forge/linux-64        Cached
  + libllvm11                11.0.1  hf817b99_0           conda-forge/linux-64        Cached
  + libnsl                    2.0.0  h7f98852_0           conda-forge/linux-64        Cached
  + libopenblas              0.3.21  pthreads_h78a6416_3  conda-forge/linux-64        Cached
  + libsqlite                3.40.0  h753d276_0           conda-forge/linux-64        Cached
  + libstdcxx-ng             12.2.0  h46fd767_19          conda-forge/linux-64        Cached
  + libuuid                  2.32.1  h7f98852_1000        conda-forge/linux-64        Cached
  + libxml2                  2.10.3  h7463322_0           conda-forge/linux-64        Cached
  + libzlib                  1.2.13  h166bdaf_4           conda-forge/linux-64        Cached
  + llvm-spirv               11.0.0  h0efe328_0           conda-forge/linux-64        Cached
  + llvmlite                 0.38.1  py39h0ddac3c_0       intel/linux-64                 5MB
  + mkl                    2022.2.1  intel_16993          intel/linux-64              Cached
  + mkl-dpcpp              2022.2.1  intel_16993          intel/linux-64              Cached
  + ncurses                     6.3  h27087fc_1           conda-forge/linux-64        Cached
  + numba                    0.55.2  py39h66db6d7_0       conda-forge/linux-64           4MB
  + numba-dpex               0.18.1  py39hfc4b9b4_50      dppy/label/dev/linux-64     Cached
  + numpy                    1.22.4  py39hc58783e_0       conda-forge/linux-64           7MB
  + openssl                   3.0.7  h0b41bf4_1           conda-forge/linux-64        Cached
  + packaging                  22.0  pyhd8ed1ab_0         conda-forge/noarch          Cached
  + pip                      22.3.1  pyhd8ed1ab_0         conda-forge/noarch          Cached
  + python                   3.9.15  hba424b6_0_cpython   conda-forge/linux-64        Cached
  + python_abi                  3.9  3_cp39               conda-forge/linux-64        Cached
  + readline                  8.1.2  h0f457ee_0           conda-forge/linux-64        Cached
  + setuptools               65.5.1  pyhd8ed1ab_0         conda-forge/noarch          Cached
  + spirv-tools              2020.5  h6bb024c_2           intel/linux-64              Cached
  + tbb                    2021.7.0  h924138e_1           conda-forge/linux-64        Cached
  + tk                       8.6.12  h27826a3_0           conda-forge/linux-64        Cached
  + tzdata                    2022g  h191b570_0           conda-forge/noarch          Cached
  + wheel                    0.38.4  pyhd8ed1ab_0         conda-forge/noarch          Cached
  + xz                        5.2.6  h166bdaf_0           conda-forge/linux-64        Cached
  + zlib                     1.2.13  h166bdaf_4           conda-forge/linux-64        Cached
```

</details>

Reply:
OK the problem was due to channel priority [which are strict in my case](https://github.com/jjerphan/dotf/blob/main/.condarc#L2).